packages/v/vulkan-hpp/xmake.lua

```diff
@@ -18,8 +18,15 @@ package("vulkan-hpp")
     add_versions("v1.3.268", "d2134fefe22279595aee73752099022222468a60")
     add_versions("v1.3.272", "e621db07719c0c1c738ad39ef400737a750bb23a")
     add_versions("v1.3.275", "1a24b015830c116632a0723f3ccfd1f06009ce12")
+    add_versions("v1.3.276", "d4b36b82236e052a5e6e4cea5fe7967d5b565ebc")
+    add_versions("v1.3.277", "c5c1994f79298543af93d7956b654bdefdfbdd26")
+    add_versions("v1.3.278", "29723f90a127ff08d9099855378162f04b4ffddd")
+    add_versions("v1.3.279", "6fb8def27290f8b87d7835457a9c68190aed9a9a")
+    add_versions("v1.3.280", "e35acfe75215116029298aebf681170559a4fe6a")
+    add_versions("v1.3.281", "88d508b32f207ba85b37fe22fe3732322d1c248d")
 
     add_configs("modules", {description = "Build with C++20 modules support.", default = false, type = "boolean"})
+    add_configs("msvc_modules", {description = "If 'modules' enabled, and you wish to use MSVC on the package, enable this to avoid a known bug of MSVC.", default = false, type = "boolean"})
```

Comment:
Why an option instead of enabling this fix automatically if toolchain is msvc and modules are enabled?